qmi/instruments/picotech/_picoscope.py

```diff
@@ -15,13 +15,13 @@
 if TYPE_CHECKING:
     from picosdk.ps3000a import ps3000a
     from picosdk.ps4000a import ps4000a
-    _ps: Union[ps3000a, ps4000a, None] = None
+    _ps: ps3000a | ps4000a | None = None
 
 else:
     _ps = None
 
 # Global dictionary for the imported library commands.
-COMMAND_DICT: Dict = dict()
+COMMAND_DICT: dict[str, Any] = dict()
```

Comment:
is it really `Any`, can't we constrain it?

Reply:
You're right, it can be a `Callable` I think. I'll try to push with that.

Reply:
You're right, it can be a Callable I think. I'll try to push with that.